Posted:    Mon Oct  3 15:33:17 2005
Subject:   first_inventory() BUG
Author:    Vital
Level:     Disembodied Head
Message: 
I am having a weird bug with first_inventory().  It appears that when it's 
called by a monster (I've never seen it bug up in other code called by a 
player) and using  this_object(), it bugs if the first_inventory() in the 
room is this_object() itself.  Trying to work around it with 
all_inventory() but that I know also has a habit of bugging up in strange 
and senseless ways.  Anyways, there any way to work around it or have it 
not bug up?  I'd use this_player() but the way I have it setup, there's no 
guarantee that the mob would be this_player() instead of a player, at 
times. 
